app/ui/exportmovieoptions.ui

```diff
@@ -60,6 +60,16 @@
         </property>
        </widget>
       </item>
+      <item>
+       <widget class="QLabel" name="unevenWidthLabel">
+        <property name="toolTip">
+         <string>The MP4 format does not support odd width. Please specify an even width or use a different file format.</string>
```

Comment:
Technically MP4 is a container format and does support odd width/height, it is the specific codec we choose to use for exporting with the MP4 container that does not support this. While I don't think users need to know the technical details like this, I think maybe "MP4 exporter" instead of "MP4 format" may be better. It could avoid confusion in the future if we add more options to MP4 export, which could potentially allow videos with an odd width/height to work.